systest/plugin/docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
       source: ../../testutil/custom_plugins
       target: /plugins
       read_only: true
-    command: /gobin/dgraph alpha --my=alpha1:7080 --zero=zero1:5080 --logtostderr
+    command: dgraph alpha --my=alpha1:7080 --zero=zero1:5080 --logtostderr
```

Comment:
Can the above /gobin bind point be removed?

Reply:
removed bind mount here too now